src/main/resources/templates/laDigitalAssister/docUploadType.html

```diff
@@ -34,7 +34,7 @@
                         helpText='',
                         content=~{::docTypes},
                         label='',
-                        ariaLabel=${userFile.get('originalFilename')})}">
+                        ariaLabel='header')}">
```

Comment:
Testing the screen reader on this screen I think it makes sense to include an aria-describedby here as well that is the original file name so that the screen reader announces the header as the label for the select input, but also reads the name of the file so that the user can know which file they are selecting a type for. 

As it is right now, they only hear the header but don't hear which file they are making a selection for.

Reply:
Like this? 

```
content=~{::docTypes},
label='',
ariaDescribedBy=${userFile.get('originalFilename')},
ariaLabel='header')}">
```